idaes/core/surrogate/tests/test_pysmo_surrogate.py

```diff
@@ -1373,9 +1371,10 @@ def test_populate_block_unisurrogate_poly(self, pysmo_surr1):
         assert blk.outputs["z1"].bounds == (None, None)
         assert isinstance(blk.pysmo_constraint, Constraint)
         assert len(blk.pysmo_constraint) == 1
-        assert str(blk.pysmo_constraint["z1"].body) == (
-            "outputs[z1] - (-75.26111111111476 - 8.815277777775934*inputs[x1] + 18.81527777777826*inputs[x2] - 2.2556956302821618e-13*(inputs[x2]*inputs[x1]))"
-        )
+        cstr = str(blk.pysmo_constraint["z1"].body).replace("outputs[z1]", "10")
```

Comment:
@eslickj If I understand this correctly you're testing the value of the evaluated expression, not the expression itself? The purpose of this test was to ensure that the pyomo block was actually generating the correct/accurate constraint on the block. There is are separate tests for evaluating the surrogate(s).

Reply:
The issue is that the test compares a string, so the floating point number have to be exactly the same.  On arm64 processors at the very least there are slight differences.  The only way I can really think to test this is to parse our the coefficients from the string and compare.  It seemed like a lot of work.

Reply:
I guess this shouldn't be too hard with regular expressions.

Reply:
You're right, it probably isn't worth the effort. We can go with that you have.